### DISCUSSION TITLE
TTYD 终端发送的响应无效

### DISCUSSION THREAD

**Discussion by lonely80** [Boos4721/openwrt#141] [Bugs/问题反馈]
已按照：https://www.vpshao123.com/archives/1032
方法进行注释，仍然如图所示。
![屏幕截图 2022-05-26 182939](https://user-images.githubusercontent.com/22154939/170470794-c90d2587-9059-4b2d-b0db-49ffb1723473.png)

Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]: ttyd 1.6.3 (libwebsockets 4.1.6-667bd40dc)
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]: tty configuration:
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]:   start command: ?
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]:   close signal: SIGHUP (1)
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]:   terminal type: xterm-256color
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]: LWS: 4.1.6-667bd40dc, loglevel 7
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]: NET CLI SRV H1 H2 WS IPV6-off
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]:    /usr/lib/libwebsockets-evlib_uv.so
Thu May 26 18:12:53 2022 daemon.notice ttyd[20474]:  Using foreign event loop...
Thu May 26 18:12:53 2022 daemon.err ttyd[20474]: ERROR on binding fd 14 to port 7681 (-1 98)
Thu May 26 18:12:53 2022 daemon.err ttyd[20474]: init server failed
Thu May 26 18:12:53 2022 daemon.err ttyd[20474]: libwebsockets vhost creation failed
Thu May 26 18:14:20 2022 daemon.notice ttyd[7400]: HTTP / - 192.168.2.161
Thu May 26 18:14:20 2022 daemon.notice ttyd[7400]: HTTP /token - 192.168.2.161
Thu May 26 18:14:21 2022 daemon.notice ttyd[7400]: WS   /ws - 192.168.2.161, clients: 1
Thu May 26 18:14:21 2022 daemon.notice ttyd[7400]: started process, pid: 20562
Thu May 26 18:15:21 2022 daemon.err ttyd[7400]: read_cb: EIO (i/o error)
Thu May 26 18:15:21 2022 daemon.notice ttyd[7400]: process exited with code 0, pid: 20562
Thu May 26 18:15:21 2022 daemon.notice ttyd[7400]: WS closed from 192.168.2.161, clients: 0

**Comment by Boos4721** [comment]
**Thu May 26 18:12:53 2022 daemon.err ttyd[20474]: ERROR on binding fd 14 to port 7681 (-1 98)**
CHECK YOUR PORT

**Reply by lonely80** [reply]
可以直接通过浏览器192.168.2.1:7861访问，但是如果通过系统→ttyd终端，就会出现截图的情况。

**Comment by beefjerky331** [comment]
把https的s去掉
